meta-balena-common/recipes-core/plymouth/plymouth_%.bbappend

```diff
@@ -17,7 +17,7 @@ do_install_append() {
     rm ${D}${nonarch_libdir}/plymouth/fade-throbber.*
     rm ${D}${nonarch_libdir}/plymouth/space-flares.*
     rm ${D}${nonarch_libdir}/plymouth/text.*
-    rm ${D}${nonarch_libdir}/plymouth/throbgress.*
+    rm ${D}${nonarch_libdir}/plymouth/throbgress.* || true
```

Comment:
This was probably not done by the conversion script, I would move to a different PR and explain why it's needed.

Reply:
Yes, it's already committed as a single patch: https://github.com/balena-os/meta-balena/pull/2368/commits/0a8a55c83704f20619bc88c6dd8fae2040db9919

Reply:
My bad, I was looking at the whole diff.